lib/features/bendpoints/BendpointMove.js

```diff
@@ -157,11 +157,9 @@ export default function BendpointMove(injector, eventBus, canvas, dragging, rule
 
       // pass hints about actual moved bendpoint
       // useful for connection/label layout
-      hints = {
-        bendpointMove: {
-          insert: insert,
-          bendpointIndex: bendpointIndex
-        }
+      hints.bendpointMove = {
```

Comment:
What additional info is being passed secretly down here? Does it make sense to be explicit and not monkey patch?

Reply:
I guess it does not matter, we monkey patch here, too: https://github.com/bpmn-io/diagram-js/pull/453/files#diff-f0a545ec36c40669962054112f362a58R170